lib/csa/ccm/cli/command.rb

```diff
@@ -58,7 +58,7 @@ def caiq2yaml(input_xlsx_file)
             return
           end
 
-          version = input_xlsx_file[/(?<=v)[0-9\.]*(?=-)/] || 'unknown'
+          version = Matrix.version_from_filepath(input_xlsx_file)
```

Comment:
Can we read the version from inside the file?

Reply:
Unfortunately, I didn't found a reliable way to do this (no dedicated cell in XSLX doc)

Reply:
Can we infer it from the title?

Reply:
In theory yes, if it's guaranteed to be there at the end of the string, or had some known format

Reply:
Can we get it from C1 ("CONSENSUS ASSESSMENTS INITIATIVE QUESTIONNAIRE v3.0.1")?

And the date of issue should be fetched from the last line of the "Change Log" worksheet:

<img width="483" alt="Screen Shot 2019-05-03 at 10 32 19 PM" src="https://user-images.githubusercontent.com/11865/57144282-527e4c00-6df3-11e9-8ec8-09a38b610e7e.png">

For CAIQ 1.1, it can be fetched from A1 ("Consensus Assessments Initiative Questionnaire v1.1"):

<img width="722" alt="Screen Shot 2019-05-03 at 10 34 10 PM" src="https://user-images.githubusercontent.com/11865/57144418-a0934f80-6df3-11e9-83e4-392e2ee2060a.png">

For CAIQ 1.0, A1 contains "Control Area".

<img width="264" alt="Screen Shot 2019-05-03 at 10 36 04 PM" src="https://user-images.githubusercontent.com/11865/57144534-d7696580-6df3-11e9-89fa-1d451042f491.png">

And A101 contains "Copyright © 2010, Cloud Security Alliance".

<img width="255" alt="Screen Shot 2019-05-03 at 10 35 50 PM" src="https://user-images.githubusercontent.com/11865/57144523-d0425780-6df3-11e9-95f3-63f647034ba4.png">

Reply:
@ronaldtse if we cannot rely on a version in the filename, what is your opinion, how should we implement `ccm-yaml 3.0.1`

Should we query all files in `resources` directory?

Reply:
In resources directory using filename is fine because we manage the files. When user uses their own CAIQ template, then we need to detect the version.

Reply:
Yep, you can check I have added `parse_version` to `Matrix` class

Reply:
@CAMOBAP795 Thanks for making `parse_version`! I don't think you need to `version` anymore here, so `Matrix.from_xlsx(input_xlsx_file)` should be sufficient.

Reply:
Done